packages/paste-core/components/side-panel/stories/components/SidePanelWithContent.tsx

```diff
@@ -0,0 +1,87 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable import/no-extraneous-dependencies */
+import { Anchor } from "@twilio-paste/anchor";
+import { Avatar } from "@twilio-paste/avatar";
+import { Box } from "@twilio-paste/box";
+import { Button } from "@twilio-paste/button";
+import { ChatComposer } from "@twilio-paste/chat-composer";
+import { ChatBubble, ChatLog, ChatMessage, ChatMessageMeta, ChatMessageMetaItem } from "@twilio-paste/chat-log";
+import { DetailText } from "@twilio-paste/detail-text";
+import { Heading } from "@twilio-paste/heading";
+import { ArtificialIntelligenceIcon } from "@twilio-paste/icons/esm/ArtificialIntelligenceIcon";
+import { MoreIcon } from "@twilio-paste/icons/esm/MoreIcon";
+import * as React from "react";
+
+import { SidePanel, SidePanelBody, SidePanelHeader, SidePanelHeaderActions } from "../../src";
+
+export const SidePanelWithContent: React.FC<React.PropsWithChildren> = () => {
+  return (
+    <SidePanel label="intelligent assistant ai bot side panel">
+      <SidePanelHeader>
+        <ArtificialIntelligenceIcon decorative size="sizeIcon50" color="colorTextIcon" />
+        <Heading as="h3" variant="heading30" marginBottom="space0">
+          Assistant
+        </Heading>
+        <SidePanelHeaderActions>
+          <Button variant="secondary_icon" size="reset" onClick={() => {}}>
+            <MoreIcon decorative={false} title="open menu" size="sizeIcon50" />
+          </Button>
+        </SidePanelHeaderActions>
+      </SidePanelHeader>
+      <SidePanelBody>
+        <Box width="100%">
```

Comment:
question(maybe): Can we add negative margin here, or treat this as a placeholder for now until we have the rest of the AI components?

Reply:
👍 I'll add this to the AI log ticket